drivers/bus/acpi/cmbatt/cmbatt.c

```diff
@@ -292,16 +294,44 @@ CmBattVerifyStaticInfo(PCMBATT_DEVICE_EXTENSION DeviceExtension,
         // FIXME: take from _BIX method: Info->CycleCount
         DeviceExtension->BifData = BifData;
```

Comment:
In one of [force-pushes](https://github.com/reactos/reactos/compare/fc33e3a760b970448694d62737e156ad001789ba..b54ca8e0c50de76ff96048d8a543c07dad0f3ec9) you've moved DesignVoltage to the top of function code, however whole BifData gets rewritten later here. Previous code accessed data from it.